content/en/docs/v3.5/quickstart.md

```diff
@@ -1,61 +1,122 @@
 ---
 title: Quickstart
 weight: 900
-description: Get etcd up and running in less than 5 minutes!
+description: Run etcd Locally Under 5 minutes!
 ---
 
-Follow these instructions to locally install, run, and test a single-member
-cluster of etcd:
+This guide walks through installing and running a single-node etcd cluster locally using pre-built binaries - no containers or cloud set up required.
 
- 1. Install etcd from pre-built binaries or from source. For details, see
-    [Install][].
+## Prerequisites
 
-    {{% alert color="warning" %}}**Important**: Ensure that you perform the last
-    step of the installation instructions to verify that `etcd` is in your path.
-    {{% /alert %}}
+Make sure you're on a supported platform before getting started:
 
- 2. Launch `etcd`:
+- Operating System: Ubuntu 22.04+ or macOS 11+. Windows users can use WSL2.
+- [Supported platforms architecture](/docs/v3.5/op-guide/)
+- Tools installed:
+  - `curl` and `tar`
+  - Internet connection to download the latest etcd [release](https://github.com/etcd-io/etcd/releases/).
```

Comment:
I don't think we want to put 'internet connection' as a pre-req. It's implied usually with every tools that need download.

Reply:
@spzala I agree. Internet is pretty much assumed these days.